src/main/java/uk/gov/hmcts/reform/managecase/config/JacksonObjectMapperConfig.java

```diff
@@ -26,6 +27,7 @@ public ObjectMapper defaultObjectMapper() {
             .registerModule(new Jdk8Module())
             .registerModule(new ParameterNamesModule(JsonCreator.Mode.PROPERTIES))
             .registerModule(new JavaTimeModule()).disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
```

Comment:
May be spring boot property like this spring.jackson.deserialization.<feature_name>=true|false  ?? 
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#howto-customize-the-jackson-objectmapper

Reply:
I did try that initially, but because we're defining our own `ObjectMapper` bean it doesn't work 😢

Reply:
aha.. that's my bad 😞  I've copied from some where during the initial setup.  I think we should probably use Spring boot auto configure feature to maximum extent and override necessary bits rather than whole ObjectMapper. 
(see JacksonAutoConfiguration , Jackson2ObjectMapperBuilder). All our current custom configuration is default now 😄  except ` @Bean
    ParameterNamesModule parameterNamesModule() {
        return new ParameterNamesModule(JsonCreator.Mode.PROPERTIES);
    }`
    
  Will raise a separate task for this.